content/en/synthetics/mobile_app_testing/_index.md

```diff
@@ -36,26 +36,30 @@ You can create mobile app tests in Datadog by navigating to [**Digital Experienc
+    1. Hover over one of the following pre-populated templates and click **View Template**. This opens a side panel displaying pre-populated configuration information, including: Test Details, Request Details, Alert Conditions, and Steps.<br /><br>
 
-## Configuration
+       {{< img src="/synthetics/browser_tests/synthetics_templates_mobile.mp4" alt="Video of Mobile Application test landing page with templates" video="true" >}}
+
+    2. Click **+Create Test** to open the configuration page, where you can review and edit the pre-populated configuration options. The fields presented are identical to those available when creating a test from scratch.
+    3. Click **Save & Quit** in the upper right hand corner when you are ready to submit your Mobile Application Test.<br /><br>
 
-Define the configuration of your mobile app test.
+- **Build a test from scratch**:
 
-1. Select a mobile application from the dropdown menu. If you haven't created one already, create a mobile application in the [Applications List section][2] on the [Synthetic Monitoring & Continuous Testing Settings page][3]. 
-2. Select a **version** or click **Always run the latest version** to use the latest version of your mobile application whenever your test is run.
-3. Add a **name** for your test.
-4. Select **environment and additional tags** that relate to your test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>`.
-4. Select the **devices** to run your test on.
-5. Set retry conditions for your test.
-6. Set the **test frequency** by clicking on basic time intervals or customizing your test frequency and **alert conditions** for your test monitor. 
-7. Enter a name for the test monitor, select a service or team member to notify, and add a message notification.
+    1. Click the **+** template, then select a mobile application from the dropdown menu. If you haven't created one already, create a mobile application in the [Applications List section][2] on the [Synthetic Monitoring & Continuous Testing Settings page][3]. 
+    1. Select a **version** or click **Always run the latest version** to use the latest version of your mobile application whenever your test is run.
+    1. Add a **name** for your test.
+    1. Select **environment and additional tags** that relate to your test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>`.
+    1. Select the **devices** to run your test on.
+    1. Set retry conditions for your test.
+    1. Set the **test frequency** by clicking on basic time intervals or customizing your test frequency and **alert conditions** for your test monitor. 
+    1. Enter a name for the test monitor, select a service or team member to notify, and add a message notification.
+    1. Click **Save & Edit Recording** when you are ready to submit your Mobile Application Test.
```

Comment:
Same note as above, with describing what the button does and flagging the capitalization :)